### DISCUSSION TITLE
Standby screen

### DISCUSSION THREAD

**Discussion by eben80** [dl9rdz/rdz_ttgo_sonde#213] [TTGO firmware]
I have this permanently running TTGO and would like some advice on how to best set up a "standby" display.
Currently I have it set to a screen with no content, which I put as the first screen in the OLED config.
The problem I have is that I'm not able to navigate away from it with the buttons. I reckon it have the timers and actions configured wrong.

How I would like it to work:
It would be this blank display by default.
I can navigate to the other screens via the buttons
After inactivity(norx), it would go back to the blank screen.

How would I set up the keyactions and timeactions to achieve this please?

**Comment by dl9rdz** [comment]
The better solution would be dedicated display on/off handling in software (including power on / power off if possible, for example the T-Beam can switch of the power completely).

Did you change the actions? If not it should work directly. Something like
@Off
timer=-1,0,0
key1action=D,#,F,W
key2action=D,#,#,#
timeaction=#,2,+

and then maybe change the next (scanner) to
@Scanner
timer=-1,0,0
key1action=D,#,F,W
key2action=D,#,#,#
timeaction=#,>,+

Maybe for the other if you use keys to go back to the scanner use
key1action=+,1,F,W  (instead of key1action=+,0,F,W)

**Comment by eben80** [comment]
Yes I read that to use the oled standby fearure when you don't have the
power supply like the beam, you have to trigger the reset on the oled to
wake.

I'd be happy to just get it working with the screens though... My main
purpose is to prevent "burn-in" which is a problem with these screens.

Thanks I'll give your suggestions a go a bit later.

**Comment by eben80** [comment]
Thank you I managed to get it working with some of your suggestions. I made the norx timer on the displays 30 seconds so it goes back to the @off display once done.
